website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,235 @@
+---
+name: 'ACL Replication for Multiple Datacenters'
+content_length: 15
+id: acl-replication
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
+primary datacenter and failover for the secondary datacenters. If the primary
+datacenter goes offline, the secondary datacenters will still have the security
+policies in place. 
+
+~> Note, each datacenter will need to have ACLs enabled, the process is outlined in the [Securing Consul with ACLs
+guide](/consul/security-networking/production-acls). This guide includes the additional ACL replication configuration for the Consul
+agents not covered in the Securing Consul with ACL guide. Additionally,
+[Basic Federation with WAN Gossip](/consul/security-networking/datacenters) is required. 
+
+In this guide, you will setup ACL replication. This is a multi-step process
+that includes:
+
+- Setting the `primary_datacenter` parameter on all Consul agents in the primary datacenter.  
+- Creating the replication token.  
+- Configuring the `primary_datacenter` parameter on all Consul agents in the secondary datacenter.  
+- Enabling token replication on the servers in the secondary datacenter.  
+- Applying the replication token to all the servers in the secondary datacenter. 
+
+This guide is intended to be completed during the initial ACL bootstrapping
+process. 
+
+-> After ACLs are enabled you must have a privileged token to complete any
+operation on either datacenter. You can use the initial
+`bootstrap` token as your privileged token.
+
+## Configure the Primary Datacenter
+
+~> Note, if your primary datacenter uses the default `datacenter` name of
```

Comment:
We should probably mention this as one of the prerequisites from the previous section unless it's required for properly WAN joining datacenters, in which case maybe we don't need to mention it at all?

Reply:
I had initially put this callout here since it's information you should know during the configuration process. Also its more for an edge case.